device-types/FireBrick/FB2900.yaml

```diff
@@ -0,0 +1,22 @@
+---
+manufacturer: FireBrick
+model: FB2900
+slug: fb2900
```

Comment:
Slugs should have the manufacturer name appended to them, in this case please append `firebrick-` to the slug